src/scss/_palette.scss

```diff
@@ -34,6 +34,9 @@ $_o-colors-default-palette-colors: join((
 		('org-b2c', #4e96eb),
 		('org-b2c-dark', #3a70af),
 		('org-b2c-light', #99c6fb),
+		
+		//brand palette
+		('ft-pink', #fcd0b1),
```

Comment:
The original issue suggests `brand-ft-pink` for clarity - reckon we need that?

Reply:
Yeah I think like Alice suggested a `brand` prefix is helpful to categories these. Though a bit confusing if you think too hard, paper etc. are also in the brand guidelines 😅 

Some documentation around it would be helpful too. Shall we add a new Brand Palette demo, and separate section to the Palette Colours README? Something like?

"`brand-ft-pink` should not be used in digital products, instead use `paper`. However since `brand-ft-pink` is used in some digital assets, such as the FT logo, there are a limited number of valid use-cases. For example  `brand-ft-pink` could be used to prevent a flash of the wrong colour as the logo loads."